JD-4-5-6/src/main/java/com/example/weekone/WeekOneApplication.java

```diff
@@ -0,0 +1,33 @@
+package com.example.weekone;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.ArrayList;
+
+@SpringBootApplication
+public class WeekOneApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(WeekOneApplication.class, args);
+        fibonacci(0, 1, 13);
+    }
+
+    public static ArrayList fibonacci(int firstNumber, int secondNumber, int lengthOfSequence){
```

Comment:
To simplify this method, since you know a valid sequence always begins with 0, 1, you can remove those from the arguments to this method and rename it to something like.

```
generateFibonacciSequence(int length)
```

Reply:
Additionally, you want to always use a generic in your return values as much as possible, but put your types in ther so this can read as

```
List<Integer> generateFibonacciSequence(int length) {
    ...
}
```

Reply:
I can make this change, but I do question. When I looked up the Fibonnacci sequence, they can start with 0 and 1, or any first two numbers. I made my method generalized, so it could be used at any starting point. In case this method was used to start at 0 for one, and say 10 for another. Do you want me to have this be hard coded to start at 0 and 1, or be capable at starting at any starting number?

Reply:
I see your point. I made mine general to start at any two numbers. I looked into more in-depth reasons why it always starts with 0 and 1. 

I found a great article that describes it with agile development with a point system. My current employer is starting to implement this with our tasks, and epics. We have a meeting tomorrow that pertains to this point system, and how we prioritize our tasks in two week sprints.

Reply:
I'm glad that the history of fibonacci was helpful outside of this context as well